es816x/src/main/scala/tech/beshu/ror/es/services/RestClientDataStreamService.scala

```diff
@@ -46,6 +47,25 @@ final class RestClientDataStreamService(client: RestClient) extends DataStreamSe
       }
   }
 
+  override protected def checkIndexLifecyclePolicyExists(policyId: NonEmptyString): Task[Boolean] = execute {
+    val policy = policyId.value
+    val request = new Request("GET", s"/_ilm/policy/$policy")
+    perform(request)
+      .flatMap {
+        case response if response.isSuccess =>
+          val policies = response.entityJson.obj.keySet
+          Task.pure(policies.contains(policy))
+        case response =>
+          failure(s"Cannot get ILM policy [$policy] - response code: ${response.statusCode}")
```

Comment:
It'd be nice to add the response body too. 
I just wonder if we should add to the message or as a debug log.